common.lic

```diff
@@ -192,8 +192,14 @@ module DRC
 
   def stow_hands
     pause 1
-    bput('stow right', 'You put') if checkright
-    bput('stow left', 'You put') if checkleft
+    stow_hand('right') if checkright
```

Comment:
I think you still need to pass in 'left' or 'right'. Sometimes you may have two of the same thing in your hand, like two ingots if you use the cutter.

If you just pass in the noun, you may stow the wrong one. So you'd need to pass in left or right, so you could stow left|right.